awkward-cpp/pyproject.toml

```diff
@@ -91,22 +91,22 @@ pytest -p no:cacheprovider \
 """]
 skip = [
     "pp*win*",
-    "pp*-manylinux_i686",
 ]
 test-skip = [
-    "*universal2:arm64",
     "pp*",
     "*musllinux*",
 ]
 build-verbosity = 1
 
 [tool.cibuildwheel.environment]
 PIP_ONLY_BINARY = "cmake,numpy"
-
-[[tool.cibuildwheel.overrides]]
-select = "cp313*"
-environment.PYO3_USE_ABI3_FORWARD_COMPATIBILITY = "1"
+UV_ONLY_BINARY = "cmake,numpy"
 
 [[tool.cibuildwheel.overrides]]
 select = "*pyodide*"
 build-frontend = {name = "build", args = ["--exports", "whole_archive"]}
+
+# Use older manylinux for Python 3.9 and 3.10
```

Comment:
This is the 3.0 default change; we are going back to the old manylinux_2014 for 3.9 and 3.10 to reduce issues if someone is running CentOS 7-like machines with older Python. NumPy moved to 2_28 too in the last release, but NumPy only supports 3.11+.